JSDemos/Demos/Map/ProvidersAndTypes/AngularJS/index.js

```diff
@@ -13,7 +13,7 @@ DemoApp.controller('DemoController', function DemoController($scope) {
         }];
     $scope.type = mapTypes[0].key;
     $scope.mapOptions = {
-        key: {
+        apiKey: {
             // NOTE: Specify your map API keys for every used provider
```

Comment:
@DokaRus can we fix this comment a bit?
```suggestion
            // Specify your API keys for each map provider: 
```

Reply:
Yep, sure